pytest_trio/plugin.py

```diff
@@ -92,7 +92,7 @@ def pytest_exception_interact(node, call, report):
 #
 # You should think of fixtures as a dependency graph: each fixtures *uses*
 # zero or more other fixtures, and is *used by* zero or more other fixtures. A
-# fixture should be setup before any of the fixtures it's used by are setup,
+# fixture should be setup before any of the fixtures it is used by are setup,
```

Comment:
in this case "it's" seems to help the readability of this awkward sentence

perhaps "A fixture should be setup before any of its dependees are setup, and torn down once all of its dependees are terminated."

Reply:
Good alternate version, I'll take it ;-)